lib/webvalve.rb

```diff
@@ -20,7 +20,7 @@ class << self
     def enabled?
       if env.in?(ALWAYS_ENABLED_ENVS)
         if ENV.key? 'WEBVALVE_ENABLED'
-          logger.warn(<<~MESSAGE)
+          logger&.warn(<<~MESSAGE)
```

Comment:
is this still an issue after your change? `Rails.logger` is not available inside an initializer? If so, i think i'd prefer that we find a way to log regardless. We could add a conditional that will use `puts` if logger is not available. 

or, perhaps we should consider making this `raise` instead of log. i'm trying to remember if there's a good reason i didn't do that the first time around.

Reply:
Yes even after my change Rails.logger is nil when this block is called. I'm pushing a little improvement now.